src/app/shared/comcol/comcol-page-browse-by/comcol-page-browse-by.component.html

```diff
@@ -3,22 +3,17 @@ <h2 class="comcol-browse-label h5">{{'browse.comcol.head' | translate}}</h2>
   <div class="d-none d-sm-block">
 
     <div class="list-group list-group-horizontal">
-      <a *ngFor="let option of allOptions"
-         class="list-group-item"
-         [routerLink]="option.routerLink"
-         [queryParams]="option.params"
-         routerLinkActive="active">{{ option.label | translate }}</a>
+      <a *ngFor="let option of allOptions" class="list-group-item" [routerLink]="option.routerLink"
```

Comment:
I think all the changes to this file can also be undone.  These are specific to Community/Collection pages again

Reply:
@GauravD2t : The changes to the `comcol-page-browse-by.component.html` are still not undone. They still are visible in the files changed tab in GitHub: https://github.com/DSpace/dspace-angular/pull/2275/files